crates/uv-cli/src/lib.rs

```diff
@@ -573,6 +573,13 @@ pub enum ProjectCommand {
     )]
     Sync(SyncArgs),
     /// Update the project's lockfile (experimental).
+    ///
+    /// If the project lockfile (`uv.lock`) does not exist, it will be created.
+    /// If a lockfile is present, its contents will be used as preferences for
+    /// the resolution.
+    ///
+    /// If there are no changes to the project's dependencies, locking will have
+    /// no effect unless the `--upgrade` flag is provided.
     #[command(
```

Comment:
Is there more to say here? @konstin / @BurntSushi

Reply:
Looks good to me.

Reply:
LGTM too